api/prd/lov/index.ts

```diff
@@ -13,16 +13,19 @@ const prdUrl: string = getConfigValue(SERVICES_PRD_COMMONDATA_API);
  */
 export async function getLovRefData(req: EnhancedRequest, res: Response) {
   // @ts-ignore
-  const { service, category, isChildRequired }  = req.query;
-  const params = new URLSearchParams({service, isChildRequired});
+  const { service, category, isChildRequired } = req.query;
+  const params = new URLSearchParams({ service, isChildRequired });
   const markupPath: string = `${prdUrl}/refdata/commondata/lov/categories/${category}?${params}`;
   try {
     // TODO the getLovFromExUI will be removed once all LoV data returns as required
     if (category === 'HearingType') {
       const exuiDefaultData = getLovFromExUI(category);
       res.status(200).send(exuiDefaultData);
+    } else if (category === 'HearingChannel') {
+      const exuiDefaultData = getLovFromExUI(category);
+      res.status(200).send(exuiDefaultData);
```

Comment:
this can be removed now, as the ONPPRS is now LIVE on demo LoV, after this is removed you will see it will returning below channels:
Key	Value_EN
INTER	In Person
TEL	Telephone
VID	Video
NA	Not in Attendance
ONPPRS	On the Papers